src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/ModelStateDictionary.cs

```diff
@@ -724,6 +757,115 @@ public PrefixEnumerable FindKeysWithPrefix(string prefix)
             return new PrefixEnumerable(this, prefix);
         }
 
+        [DebuggerDisplay("SubKey={SubKey}, Key={Key}, Visible ? ValidationState : \"hidden\"")]
```

Comment:
Right, that's why the debugger will display exactly `... Visible ? ValidationState : \"hidden\"`. Need to add the curly braces.